### PR TITLE
Do not disable in production

### LINKED ISSUE #262: Remove production build resource filtering behaviour
At the moment the addon removes all of the resources from the build when doing a production build. Given that at the moment the default `npm build` command is a production build, it means that the default deploy experience is a broken app, since it will try to render a non-existent `<WelcomePage>` component.

### PR DESCRIPTION
**Pull request by rwwagner90**
This makes things not disabled in production anymore.

Closes #262

### CODE REVIEW

**Comment by locks** [comment]
I'd like @rwjblue to give a once over before we merge!

File: index.js [hunk 1]
```diff
@@ -3,51 +3,18 @@
...
-    this._welcomeConfig = app.options['ember-welcome-page'] || {};
-
-    if (this._isDisabled()) { return; }
-
     app.import('vendor/welcome-page.css');
   },
 
-  config: function() {
-
+  config() {
```
Comment (rwjblue) [inline index.js:12]: We should remove this also, any MU support code should be released before the next major.

Comment (rwwagner90) [reply index.js:12]: I had initially removed all MU code, but @locks wanted me to remove those changes to make the changelog cleaner and do it in another PR.

**Comment by rwwagner90** [comment]
@locks @rwjblue what's the status here?

**Comment by rwjblue** [comment]
The convo in https://github.com/ember-cli/ember-welcome-page/issues/262 is current, and AFAICT the status is unchanged.

**Comment by rwwagner90** [comment]
@locks @rwjblue I'm going to close this if we don't want it. Just let me know.

**Pull request closed without merging**